.github/workflows/ci.yaml

```diff
@@ -42,11 +42,13 @@ jobs:
           coverage report -m --skip-covered
 
       - name: Check coverage report
-        if: github.ref != 'refs/heads/main'
+        # Disable coverage checking from forks until there is a secure fix for:
+        # https://github.com/orgoro/coverage/issues/259
+        if: github.ref != 'refs/heads/main' && ${{ ! github.event.pull_request.head.repo.fork }}
```

Comment:
This is unrelated, but I noticed this problem when trying to to land https://github.com/smart-on-fhir/client-py/pull/109 -- coverage fails on fork branches  :frowning:

Reply:
oh fascinating - i guess that should probably be something we do elsewhere.